### PR TITLE
Reworked specialized medkit contents

### PR DESCRIPTION
**Pull request by Superlagg**
[FEATURE]
## About the PR 

Changes the contents of toxin, brain, and oxy medkits, adds a new "crit medkit", and adds perfluorodecalin and pentetic acid auto-injectors.

Toxic medkits now contain:
 - 1 calomel syringe
 - 2 charcoal pills
 - 2 potassium-iodide pills
 - 1 epinephrine auto-injector
 - 1 medscanner

Brain medkits now contain:
 - 3 mannitol pills
 - 2 mannitol autoinjectors
 - 1 epinephrine auto-injector
 - 1 medscanner

Oxy medkits now contain:
 - 3 salbutamol pills
 - 2 salbutamol auto-injectors
 - 1 epinephrine auto-injector
 - 1 medscanner

The new crit medkits contain:
 - 1 brute auto-mender
 - 1 burn auto-mender
 - 1 atropine auto-injector
 - 1 epinephrine auto-injector
 - 1 saline-glucose auto-injector
 - 1 synaptizine auto-injector
 - 1 upgraded medscanner
It also uses the old-y medkit color!

The crit medkit is currently only found via the QM medkit crate, and through purchase from D.O.C. Base price is 2000 big ones.

Also, the pent injector can be found through D.O.C. and presumably any other medical trader. This is just so that it can be obtained, like, at all.

## Why's this needed? 

As described in https://forum.ss13.co/showthread.php?tid=14588 , three of the medkits really could use some love. I also feel like every medkit should have an epi injector!

## Changelog

```
(u)Superlagg:
(*)Tox, brain, and oxy medkits now have more useful auto-injectors!
(+)Added a new emergency medkit specifically for deep crit. Ask your local QM or salesbot today!
```

### CODE REVIEW

**Comment by ZeWaka** [comment]
I don't see any way to obtain the crit medkit?

**Comment by Superlagg** [comment]
> I don't see any way to obtain the crit medkit?

I haven't added any way to obtain it. I could try adding it to a QM purchase or the MD's locker (though their locker is already crammed full of fun toys!). Any thoughts?

**Comment by Carbadox** [comment]
how about adding it to MedKit locker and crate?

**Comment by ZeWaka** [comment]
> 
> 
> how about adding it to MedKit locker and crate?

On some maps, these aren't used and are manually placed on tables, so if this is done then it would have to be compensated for on those maps.

**Comment by Gerhazo** [comment]
I disagree with the toxin kit contents, or more specifically one of them.
Pentetic acid is superbly effective and practically a in most cases better version of calomel, but also in very limited supply (there's only a single bottle in a restricted locker that's often only accessible to the medical director).

I think this dynamic of having a powerful item but also requiring department assistance (chemistry or cargo bay) in order to acquire more is fun and I'd probably want to see it exist for more interactions.

Putting it in a regular anti-toxin kit will make it a common place item, which I think will both subtract from my forementioned interaction as well as affect gameplay indirectly in that it'll make treating poisoning and radiation very significantly easier.

**Comment by Cthucky** [comment]
Pentetic acid is way too good to be included in something as common as a medkit.  As to Gerhazo's suggestion of calomel, I considered it when making the suggestion on the forums but I had recently run into people trying to use it to treat tox damage and I figured an antitoxin medkit probably doesn't need anything that will make you need an antitoxin medkit.

**Comment by Sovexe** [comment]
Crit should maybe contain saline glucose

Since you are basically always in shock when in  crit

**Comment by UrsulaMejor** [comment]
The CritKit could be sold by D.O.C. for some amount of money; it seems like the kind of thing he'd have.

**Comment by Superlagg** [comment]
Okay, I moved the pent injector to the crit kit and replaced the one in the tox kit with a calomel syringe.

Also, the crit kit can now be ordered in the restricted medical crate, or purchased from D.O.C.

**Comment by Tarmunora** [comment]
I'd personally replace the pent injector in the crit-kit with salglu. Pent flushes other chems, which is rarely ideal in a crit situation

**Comment by Superlagg** [comment]
> I'd personally replace the pent injector in the crit-kit with salglu. Pent flushes other chems, which is rarely ideal in a crit situation, and emergency toxin-healing is a rare case

Heck, forgot it did that! Okay, swapped out the pent injector for a saline injector, and moved the pent injector to the restricted meds QM crate.

Also, I moved the crit kit to the assorted medkit QM crate.

Should I give the pent injector to D.O.C.?

File: code/obj/item/storage/med_chem.dm [hunk 1]
```diff
@@ -71,29 +71,48 @@
...
+/obj/item/storage/firstaid/crit
+	name = "emergency critical-condition first aid"
+	icon_state = "berserk1"
+	item_state = "berserk1"
+	desc = "An all-in-one emergency crash-kit designed to bring a critically wounded patient back from the brink of death. Contains cardiac stimulants, cerebral anti-edemics, and blood-oxygenation agents, all in convenient auto-injector form. Also contains an upgraded health analyzer to determine the health of the patient."
+	kit_styles = list("berserk1", "berserk2", "berserk3")
+	spawn_contents = list(/obj/item/reagent_containers/emergency_injector/atropine = 1,\
+	/obj/item/reagent_containers/emergency_injector/perf = 1,\
+	/obj/item/reagent_containers/emergency_injector/saline,\
+	/obj/item/reagent_containers/emergency_injector/mannitol = 2,\
```
Comment (Tarmunora) [inline code/obj/item/storage/med_chem.dm:113]: in light of recent changes, replacing the 2 mannitol injectors with a synaptizine injector and cutting the perf (atro+epi handles oxy well enough) might be better. Perhaps fill the space with one of each styptic and sulf patches

File: code/datums/supply_packs.dm [hunk 2]
```diff
@@ -702,7 +703,8 @@
 					/obj/item/reagent_containers/glass/bottle/pfd = 2,
 					/obj/item/reagent_containers/glass/bottle/pentetic,
 					/obj/item/reagent_containers/glass/bottle/haloperidol,
-					/obj/item/reagent_containers/glass/bottle/ether)
+					/obj/item/reagent_containers/glass/bottle/ether,
+					/obj/item/reagent_containers/emergency_injector/pentetic_acid = 2)
```
Comment (Tarmunora) [inline code/datums/supply_packs.dm:707]: Not a huge fan of this, as there's a bottle of pent in here anyways, and the rest of the crate is all bottles

File: code/obj/item/storage/med_chem.dm (hunk 1 above)
Comment (Superlagg) [reply code/obj/item/storage/med_chem.dm]: Done! Though instead of patches, they get auto-menders.

File: code/datums/supply_packs.dm (hunk 2 above)
Comment (Superlagg) [reply code/datums/supply_packs.dm]: Done! Moved the pent injector to D.O.C. and cleared it out of the crate!

**Comment by Superlagg** [comment]
Swapped the menders in the crit kit with a perf and mannitol injector, added the perf injector to DOC, and swapped the calomel syringe with a charcoal injector.

**Comment by TDHooligan** [comment]
These look a lot better. I personally think there may be even be some use to throw one or two crit kits in around accident prone parts of the station for a crafty assistant to pick up. They aren't quite as valuable as regular first aid kits without the menders, so they could be a useful way to introduce new players to chems that fix people in crit.

**Comment by pali6** [comment]
Someone better versed in medbay and balance please look at this.

File: code/datums/commodity.dm [hunk 3]
```diff
@@ -1893,6 +1893,30 @@
 	upperfluc = 100
 	lowerfluc = -100
 
+/datum/commodity/medical/firstaidC
+	comname = "First Aid Kit - Critical"
+	comtype = /obj/item/storage/firstaid/crit
+	price = 2000
+	baseprice = 2000
```
Comment (Tarmunora) [inline code/datums/commodity.dm:1900]: I'd raise the price on this a bit and remove it from the QM crate to reinforce status as a high-end luxury medical item - maybe add as a military-locked debris crate loot option?

**Comment by ZeWaka** [comment]
I'd like to merge this with the caveat of the above comment.

**Comment by Superlagg** [comment]
Removed crit kit from QM medkit box, increased price to 10k, and made it spawn alongside the phaser in military crate.

**Pull request merged**